### PR TITLE
Status

### PR DESCRIPTION
**Pull request by GrishaDev**
field alive changed to status

### CODE REVIEW

File: src/person/person.repository.ts [hunk 1]
```diff
@@ -19,7 +19,15 @@ export class PersonRepository extends RepositoryBase<IPerson> {
    */
   getPersonsByQuery(queryFields: any = {}, populate?: any, select?: any): Promise<IPerson[]> {
     const cond = {};
-    if (!(queryFields.alsoDead && queryFields.alsoDead === 'true')) cond['alive'] = 'true';
+
+    let query;
+    if (queryFields.status) {
+      query = queryFields.status.toLowerCase();
+    }
```
Comment (eladex) [inline src/person/person.repository.ts]: can be:
`const query = queryFields.status ? queryFields.status.toLowerCase() : ''`

File: src/person/person.repository.ts [hunk 2]
```diff
@@ -19,7 +19,15 @@ export class PersonRepository extends RepositoryBase<IPerson> {
...
+    if (query !== consts.STATUS[3]) {
+      cond['status'] = consts.STATUS.indexOf(query) > -1 ? query : consts.STATUS[0];
```
Comment (eladex) [inline src/person/person.repository.ts]: use `Array.includes()`

File: src/person/person.model.ts [hunk 3]
```diff
@@ -112,16 +112,16 @@ export const PersonSchema = new mongoose.Schema(
       type: String,
       enum: consts.CURRENT_UNIT,
     },
-    alive: {
-      type: Boolean,
-      default: true,
+    status: {
+      type: String,
+      default: consts.STATUS[0],
```
Comment (eladex) [inline src/person/person.model.ts]: add enum to allow only legal statuses

File: src/person/person.repository.ts (hunk 2 above)
Comment (GrishaDev) [reply src/person/person.repository.ts]: there is some problem with includes() and es version used by kartoffel

Comment (GrishaDev) [reply src/person/person.repository.ts]: nvm

**Pull request merged**